src/lsp-client/LSPEditorWrapper.cpp

```diff
@@ -466,6 +466,16 @@ LSPEditorWrapper::PrevCallTip()
 	fCallTip.PrevCallTip();
 }
 
+void
+LSPEditorWrapper::RequestDocumentSymbols()
+{
+	if (!fLSPProjectWrapper || !fEditor)
```

Comment:
we should check it explicitly i.e. == nullptr

Reply:
Done